auth-web/package.json

```diff
@@ -30,7 +30,7 @@
     "moment": "^2.29.4",
     "regenerator-runtime": "^0.13.3",
     "register-service-worker": "^1.6.2",
-    "sbc-common-components": "3.0.1",
+    "sbc-common-components": "3.0.3",
```

Comment:
Maybe bump up line 3 as well (even though that version is not exposed yet).

Reply:
should be good